amida-tech/indaba-client
# Bug/INBA-457 Edit User Groups

#### What does this PR do?
Fixes group editing. Makes a call to the backend to update the existing group after clicking Save in a user group edit modal.

#### Related JIRA tickets:
[INBA-457](https://jira.amida-tech.com/browse/INBA-457)

#### How should this be manually tested?
1. Click on a group name in the group list on the project management page.
1. Add or remove users from the group in the modal
1. Check that the backend updates the UserGroups table with the new group membership, and that the frontend updates the users and project with the new membership in redux.

#### Background/Context
This requires the backend to have https://github.com/amida-tech/greyscale/pull/402

#### Screenshots (if appropriate):
